frontend/src/metabase/visualizations/visualizations/CartesianChart/events.ts

```diff
@@ -490,15 +480,23 @@ export const getSeriesOnlyTooltipModel = (
 
   const seriesRows: EChartsTooltipRow[] = chartModel.seriesModels
     .filter(seriesModel => seriesModel.visible)
-    .map((seriesModel, seriesIndex) => {
+    .map(seriesModel => {
       const isHoveredSeries = seriesModel.dataKey === hoveredSeries.dataKey;
       const isFocused = isHoveredSeries && chartModel.seriesModels.length > 1;
 
-      const prevValue = computeDiffWithPreviousPeriod(
-        chartModel,
-        seriesIndex,
-        dataIndex,
-      );
+      const value =
+        seriesModel.dataKey === OTHER_DATA_KEY
+          ? getAggregatedOtherSeriesValue(
```

Comment:
What if we could have this series values right in `chartModel.dataset[OTHER_DATA_KEY]`? It seems that would simplify the code a bit, what do you think? It does not block this PR to get merged

Reply:
Good catch! I ended up reusing the value from `chartModel.transformedDataset` [`5f7da16` (#48265)](https://github.com/metabase/metabase/pull/48265/commits/5f7da162f0cbeeae9274bf221d8d9e5406fbd406)